### ISSUE TITLE
After create-react-native-app and getstorybook I doesn't see any components

### ISSUE THREAD

**Issue by master-7** [storybooks/storybook#1857]
package.json
```
{
  "name": "Locator",
  "version": "0.1.0",
  "private": true,
  "devDependencies": {
    "babel-preset-react-native-stage-0": "^1.0.1",
    "jest-expo": "~20.0.0",
    "prop-types": "^15.5.10",
    "react-dom": "16.0.0-alpha.12",
    "react-native-scripts": "1.3.1",
    "react-test-renderer": "16.0.0-alpha.12",
    "@storybook/react-native": "^3.2.8",
    "@storybook/addon-actions": "^3.2.6",
    "@storybook/addon-links": "^3.2.6"
  },
  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
  "scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "node node_modules/jest/bin/jest.js --watch",
    "storybook": "storybook start -p 7007"
  },
  "jest": {
    "preset": "jest-expo"
  },
  "dependencies": {
    "expo": "^20.0.0",
    "react": "16.0.0-alpha.12",
    "react-native": "^0.47.0"
  }
}
```

.babelrc

```
{
  "presets": ["babel-preset-expo", "react-native-stage-0"],
  "env": {
    "development": {
      "plugins": ["transform-react-jsx-source"]
    }
  }
}

```

<img width="1215" alt="screen shot 2017-09-17 at 00 12 02" src="https://user-images.githubusercontent.com/7540752/30516063-e922d760-9b3c-11e7-8bce-512c3a447513.png">

**Comment by master-7** [comment]
If I run `yarn ios` I see
<img width="323" alt="screen shot 2017-09-17 at 00 14 54" src="https://user-images.githubusercontent.com/7540752/30516076-3f8c0fcc-9b3d-11e7-9a44-3495e0b1f27b.png">

**Comment by danielduan** [comment]
You need to have storybook running in the background as well:
https://github.com/storybooks/storybook/tree/master/app/react-native#start-storybook

**Comment by ximenean** [comment]
I get this as well (not seeing the default stories in the web browser). Also replicated on a sample project created with react-native init. I'm using RN 0.49.3

**Comment by kmdupr33** [comment]
I'm having the same problem with RN 0.49.5

**Comment by ximenean** [comment]
I think it's related to the changes in React native 0.49 which replaces the platform-specific `index.ios.js` and `index.android.js` with a single `index.js`. I found that if I added an `index.js` in the `storybook` folder it worked as expected.

There is a PR ( #1971 ) for RN 0.49 support but it looks to update dependencies only. Would it be worth submitting a PR for this?

**Comment by ximenean** [comment]
I've created #2176

**Comment by danielduan** [comment]
published in `3.2.13`
